HCaptcha/Classes/HCaptcha.swift

```diff
@@ -58,6 +34,36 @@ public class HCaptcha {
         /// The base url to be used to resolve relative URLs in the webview
         let baseURL: URL
 
+        /// Custom supplied challenge data
+        let rqdata: String?
+
+        /// Enable / Disable sentry error reporting.
+        let sentry: Bool
+
+        /// The url of api.js
+        /// Default: https://hcaptcha.com/1/api.js
+        let apiEndpoint: URL
+
+        /// Point hCaptcha JS Ajax Requests to alternative API Endpoint.
+        /// Default: https://hcaptcha.com
+        let endpoint: URL
+
+        /// Point hCaptcha Bug Reporting Request to alternative API Endpoint.
+        /// Default: https://accounts.hcaptcha.com
+        let reportapi: URL
+
+        /// Points loaded hCaptcha assets to a user defined asset location, used for proxies.
+        /// Default: https://assets.hcaptcha.com
+        let assethost: URL
+
+        /// Points loaded hCaptcha challenge images to a user defined image location, used for proxies.
+        /// Default: https://imgs.hcaptcha.com
+        let imghost: URL
+
+        /// Platform identifier where SDK works
```

Comment:
@e271828- I'm not sure about comment, feel free to suggest something more applicable